sphinxcontrib/spelling/asset.py

```diff
@@ -0,0 +1,29 @@
+# encoding: utf-8
+#
+# Copyright (c) 2020 Doug Hellmann.  All rights reserved.
+#
+"""Asset collector for additional spelling terms
+"""
```

Comment:
PEP257 requires a trailing period here
```suggestion
"""Asset collector for additional spelling terms."""
```